src/textures/Texture.js

```diff
@@ -344,7 +344,7 @@ class Texture extends EventDispatcher {
 		 * @readonly
 		 * @default false
 		 */
-		this.isTextureArray = false;
+		this.isArrayTexture = image && image.depth && image.depth > 1 ? true : false;
```

Comment:
Is it safe to assume all textures with a depth > 1 should be array textures? What if a texture is used as a 3D texture? In this case it has a depth but it shouldn't be classified as a array texture.

Reply:
`isCubeTexture `will override `isArrayTexture`, I would update the Renderer with something like this then:
```js
if (this.isArrayTexture) {

    if (this.isCubeTexture) {
    }
}
```
and to be extra safe we could just set `isArrayTexture` to false in the `isCubeTexture` Class.